tests/vectordbs/test_qdrant.py

```diff
@@ -0,0 +1,112 @@
+"""Tests for verifying if the `qdrant` vector DB server is up and running."""
+
+import docker
+import numpy as np
+import pytest
+from qdrant_client import QdrantClient
+from qdrant_client.models import Distance, PointStruct, VectorParams
+
+import inatinqperf
+
+
+@pytest.fixture(scope="module", autouse=True)
+def container_fixture():
+    """Start the docker container with the vector DB."""
+    client = docker.from_env()
+    container = client.containers.run(
+        "qdrant/qdrant",
+        ports={"6333": "6333"},
+        remove=True,
+        detach=True,  # enabled so we get back a Container object
+    )
+
+    yield container
+    container.stop()
```

Comment:
I'm not familiar with this api, but should the test container be deleted after it is stopped?